sdk/src/main/java/io/radar/sdk/RadarHuaweiLocationClient.kt

```diff
@@ -0,0 +1,135 @@
+package io.radar.sdk
+
+import android.annotation.SuppressLint
+import android.app.PendingIntent
+import android.content.Context
+import android.location.Location
+import android.os.Looper
+import com.huawei.hms.location.*
```

Comment:
better for compilation speed to not do the wildcard imports

Reply:
Fixed (and updated my Android Studio settings to avoid in the future)